web/satellite/src/components/accessGrants/modals/CreateFormModal.vue

```diff
@@ -40,7 +40,7 @@
                         @mouseover="toggleTooltipHover('access','over')"
                         @mouseleave="toggleTooltipHover('access','leave')"
                     >
-                        <span class="tooltip-text">Keys to upload, delete, and view your project's data.  <a class="tooltip-link" href="https://storj-labs.gitbook.io/dcs/concepts/access/access-grants" target="_blank" rel="noreferrer noopener">Learn More</a></span>
+                        <span class="tooltip-text">Keys to upload, delete, and view your project's data.  <a class="tooltip-link" href="https://storj-labs.gitbook.io/dcs/concepts/access/access-grants" target="_blank" rel="noreferrer noopener" @click="trackPageVisit('https://storj-labs.gitbook.io/dcs/concepts/access/access-grants')">Learn More</a></span>
```

Comment:
nit - this should be https://docs.storj.io/dcs/concepts/access/access-grants

Reply:
updated the link